cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -22,6 +22,8 @@ import (
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// MasterConfiguration contains a list of elements which are making up master's
```

Comment:
"MasterConfiguration contains a list of elements which make up the master's"

Reply:
done